### ISSUE TITLE
Use Case: 3rd Party Content for Secure Sites

### ISSUE THREAD

**Issue by ChadKillingsworth** [WICG/portals#114]
I just wanted to document my use case and re-enforce the utility that portals would bring.

I develop an online banking platform for hundreds of US based financial institutions. We're planning on allowing 3rd parties to provide rich web apps embedded in our site via sandboxed iframes. However navigation will take the user to the 3rd party site. After seeing portals, it seems like a much better experience and highly preferable to a standard iframe.

Since this is outside of the standard link aggregation site use case, I wanted to re-enforce the utility in other types of sites.

**Comment by lucasgadani** [comment]
Thanks for the feedback! Yes, this is one of the main use cases we have considered since the start of the design. Basically, we want to be able to provide a user experience where the user transitions from a page, to a payment processor, back to the page, with smooth transitions, but at the same time being able to verify the origin of the payment processor through the URL bar, which isn't possible with iframes.

**Comment by sucrose** [comment]
@ChadKillingsworth Your use case is to use portals to transition users away from your platform?

**Comment by ChadKillingsworth** [comment]
Yes. Imagine a portal embedded in your online banking site that shows investment accounts at a brokerage. Or invoices and payments from a small business budget site. Interaction with these sites would indeed take you away from my platform.

This allows a financial institution to partner with another provider and provide rich interactive launch points within my platform.

**Comment by ChadKillingsworth** [comment]
History / Back button support would be important for us. The version of portals in Chrome Canary doesn't seem to work with the back button. I'm not sure that's intentional though.

**Comment by lucasgadani** [comment]
@ChadKillingsworth This is a known issue and it's tracked on #19 .
